src/accidental.ts

```diff
@@ -15,7 +15,7 @@ import { Category, isAccidental, isGraceNote, isGraceNoteGroup, isStaveNote } fr
 import { defined, log } from './util';
 import { Voice } from './voice';
 
-type StaveLineAccidentalLayoutMetrics = {
+export type StaveLineAccidentalLayoutMetrics = {
```

Comment:
I exported this type since the doc generation complained about it.

I agree that checkCollision should be private. I will rename it to `#checkCollision`, to take advantage of JavaScript's private method syntax: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields